patches/server/0570-Missing-Entity-API.patch

```diff
@@ -249,6 +249,25 @@ index 37da58385ecf40baeb4665a0d9c262da03c01763..4fa1a12e7e5c23e1d902b59f2a315eac
  
      public Llama(EntityType<? extends Llama> type, Level world) {
          super(type, world);
++        this.maxDomestication = 30; // Paper
```

Comment:
needs identifying comment as to what is happening here.

// Paper - Missing entity api - configure max temper instead of a hardcoded value